<launch>
  <!-- ORCA Planner Node -->
  <node name="orca_test_node" pkg="orca_planner" type="orca_test_node" output="screen">
    <!-- Parameters -->
    <param name="goal_dist_tolerance" value="0.2"/>
    <param name="rotate_tolerance" value="0.5"/>
    <param name="uav_name" value="uav"/>
    <param name="uav_num" value="3"/>
    <param name="agent_number" value="3"/>
    <param name="agent_id" value="1"/>
    <param name="max_v" value="0.5"/>
    <param name="min_v" value="0.0"/>
    <param name="max_v_inc" value="0.5"/>
    <param name="max_w" value="1.57"/>
    <param name="min_w" value="0.0"/>
    <param name="max_w_inc" value="1.57"/>
    <param name="neighbor_dist" value="3.0"/>
    <param name="time_horizon" value="5.0"/>
    <param name="time_horizon_obst" value="3.0"/>
    <param name="radius" value="0.2"/>
    <param name="max_neighbors" value="10"/>
    <param name="controller_frequency" value="10.0"/>
  </node>
</launch>